.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "singleQuote": true,
+  "semi": true,
+  "trailingComma": "all",
+  "printWidth": 80,
+  "tabWidth": 2
```

Comment:
Why tab instead of whitespaces?

Reply:
it's just a common setting with prettier, defines that it uses 2 spaces for indentation.